corehq/util/test_utils.py

```diff
@@ -60,6 +60,27 @@ def trap_extra_setup(*exceptions, **kw):
         raise
 
 
+def softer_assert(func=None):
```

Comment:
On second thought, that changes the semantics of the decorator. Instead of 
```py
@softer_assert
def test_foo():
    ...
```
it has to be
```py
@softer_assert()
def test_foo():
    ...
```
Would have to use crazy `__new__` tricks in `ContextDecorator` to get around that. If you don't mind I'm going to keep it the way it is.